src/main/java/com/tabnine/Initializer.java

```diff
@@ -37,4 +43,16 @@ public void appStarting(@Nullable Project projectFromCommandLine) {
         binaryNotificationsLifecycle.poll();
         binaryPromotionStatusBarLifecycle.poll();
     }
+
+    private void initLogger() {
+        Sentry.init();
+        Sentry.configureScope(scope -> {
+            scope.setTag("ide", ApplicationInfo.getInstance().getVersionName());
+            scope.setTag("ideVersion", ApplicationInfo.getInstance().getFullVersion());
+            scope.setTag("os", System.getProperty("os.name"));
+            scope.setTag("channel", Config.CHANNEL);
+        });
+        org.apache.log4j.Logger rootLogger = LogManager.getRootLogger();
```

Comment:
Hmmm.... won't this send us all sorts of errors from intellij too?
Maybe add it under "com.tabnine" or something like that?

Reply:
oh, you are right, i added a filter https://github.com/codota/tabnine-intellij/blob/ec999dc55e260d0fa68dfff2cb4487487976b98e/src/main/java/com/tabnine/logging/TabnineFilter.java#L6
do you know a better option for this?